apps/console/src/extensions/configs/models/analytics.ts

```diff
@@ -20,7 +20,7 @@ export interface AnalyticsConfig {
     EventPublisherExtension: {
         compute: (computation: () => void) => void;
         init: () => void;
-        publish: (eventId: string, customProperties?: { [key: string]: string | Record<string, unknown> |
+        publish: (eventId: string, _customProperties?: { [key: string]: string | Record<string, unknown> |
```

Comment:
We don't need `_` here since this is only an interface.

Reply:
Fixed